src/gui/optionsdlg.cpp

```diff
@@ -95,7 +99,13 @@ OptionsDialog::OptionsDialog(QWidget *parent)
 #endif
     m_ui->tabSelection->item(TAB_ADVANCED)->setIcon(GuiIconProvider::instance()->getIcon("preferences-other"));
     for (int i = 0; i < m_ui->tabSelection->count(); ++i) {
-        m_ui->tabSelection->item(i)->setSizeHint(QSize(std::numeric_limits<int>::max(), 64));  // uniform size for all icons
+        // uniform size for all icons
+#ifdef Q_OS_MAC
+        // Smaller size for mac due to rendering specifics to avoid vertical scroll
```

Comment:
Hm, but will this be fine? I do not see many reasons to increase the total height, and I am not sure if reducing the size is ok on all the platforms.